### ISSUE TITLE
[css-fonts] Computed value of font-feature-settings

### ISSUE THREAD

**Issue by jfkthame** [w3c/csswg-drafts#8296]
The current text of CSS Fonts 4 seems to contradict itself here.

On the one hand, [6.12 says](https://www.w3.org/TR/css-fonts-4/#font-feature-settings-prop)

>  Computed value:  as specified

Suppose the specified value is
```
font-feature-settings: "abcd", "spqr", "abcd" 2;
```

Note that the "abcd" feature has been specified twice, first with the implied value of `1`, and later with an explicit value of `2`.

So what is the computed  value of `font-feature-settings`? If it's "as specified", then it should be exactly this list of three features, including both entries for "abcd".

However, the text goes on to say, a few paragraphs later,

> The computed value of font-feature-settings is a map, so any duplicates in the specified value must not be preserved. If the same axis name appears more than once, the value associated with the last appearance supersedes any previous value for that axis.

Currently, of the browsers I've tried, only Safari appears to do this, returning `"spqr", "abcd" 2` from getComputedStyle; both Firefox and Chrome return the list "as specified" including the first (superseded) occurrence of `"abcd"`.

We should confirm which behavior is wanted ("as specified" or "with duplicates removed"), and adjust the spec accordingly. (My inclination is to require the duplicate-removal behavior, as implemented in Safari, so that would imply Firefox and Chrome should fix their bugs here.)

(Note that the similarly-structured [`font-variation-settings` property](https://www.w3.org/TR/css-fonts-4/#font-variation-settings-def) defines its computed value differently:

>  Computed value:  the keyword 'normal' or a list, each item a string paired with a number

It then has similar text confirming that duplicates should be removed:

> If the same axis name appears more than once, the value associated with the last appearance supersedes any previous value for that axis. This deduplication is observable by accessing the computed value of this property.

So maybe a similar computed value description should be used for `font-feature-settings`, instead of "as specified".)

**Comment by svgeesus** [comment]
I agree that dups should be removed, as the spec says, so the fix here is to change "as specified" to "as specified, with duplicates removed as described below".

Does "a list" imply that the as-specified order is retained?

**Comment by jfkthame** [comment]
By the way, when you get to editing this, there's another issue with the existing text where it says:

> However, the text goes on to say, a few paragraphs later,
> 
> > The computed value of font-feature-settings is a map, so any duplicates in the specified value must not be preserved. If the same axis name appears more than once, the value associated with the last appearance supersedes any previous value for that axis.

This looks like it was copy/pasted from the corresponding text about font-variation-settings, but it's not appropriate to refer to "...the same axis name..." and "...value for that axis" here. The term "axis" is specific to _variations_, not _features_.

So this should read something like "...the same feature tag..." and "...value for that feature" instead.
